cmd/globals.go

```diff
@@ -170,5 +175,30 @@ func setGlobalsFromContext(ctx *cli.Context) error {
 		}
 	}
 
+	dnsEntries := ctx.StringSlice("resolve")
+	if len(dnsEntries) > 0 {
+		globalResolvers = make(map[string]netip.Addr, len(dnsEntries))
+
+		// Each entry is a HOST[:PORT]:IP pair. This matches cURL's syntax.
+		for _, e := range dnsEntries {
+			i := strings.LastIndexByte(e, ':')
```

Comment:
Why the last?  Wouldn't that break on an IPv6? Wouldn't you enter that like this: `minio.local:2001:0db8:85a3:0000:0000:8a2e:0370:7334`.  Hmmm. That makes the optional port clunky.

Maybe splitting on `=` would make it clearer. `minio.local:9000=2001:0db8:85a3:0000:0000:8a2e:0370:7334`.

With `=` it would be a lot easier easier to parse - and I don't see either side having an `=` char - unlike other places ;)

Reply:
That is true. 👍 I wanted to stick to `curl` for being able to copy paste things. (useful for debugging instead of having to adjust things)
However, no strong opinion. We can also use `=`.